cardano-cli/test/cardano-cli-test-lib/Test/Cardano/CLI/Aeson.hs

```diff
@@ -0,0 +1,114 @@
+module Test.Cardano.CLI.Aeson (
+    assertHasKeys,
+    assertHasMappings,
+  ) where
+
+import           Control.Monad (forM_)
+import           Control.Monad.IO.Class
+import           Data.Aeson hiding (pairs)
+import qualified Data.Aeson.Key as Aeson
+import qualified Data.Aeson.KeyMap as Aeson.KeyMap
+import qualified Data.ByteString.Lazy as LBS
+import           Data.Text (Text)
+import qualified Data.Text as T
+
+import           Hedgehog
+import qualified Hedgehog as H
+import qualified Hedgehog.Extras as H
+
+{- HLINT ignore "Use uncurry" -}
+
+-- | @assertHasKeys keys path@ succeeds if @path@ is a file containing a JSON object
+-- whose keys is a superset of @keys@.
+--
+-- For example. if @path@ contains @"{ "a":0, "b":1.0, "c": "foo"}"@,
+-- @hasKeys ["b", "a"] path@ succeeds.
+assertHasKeys :: ()
+  => MonadTest m
+  => MonadIO m
+  => [Text]
+  -> FilePath
+  -> m ()
+assertHasKeys keys jsonFile = do
```

Comment:
```suggestion
assertHasKeys :: ()
  => HasCallStack
  => MonadTest m
  => MonadIO m
  => [Text]
  -> FilePath
  -> m ()
assertHasKeys keys jsonFile = withFrozenCallStack $ do
```
otherwise hedgehog will report an error from this module instead of the test suite

This also applies to other functions in this module.